config/initializers/omniauth.rb

```diff
@@ -40,13 +40,19 @@ def callback_phase
         fail!(:timeout, e)
       rescue ::SocketError => e
         fail!(:failed_to_connect, e)
+      rescue Rack::OAuth2::Client::Error => e
+        fail!(:invalid_client, e)
```

Comment:
how about `:client_error` to match the caught exception? I wonder if invalid client could cause confusion if we hit the same exception for different reasons

p.s. I'm not really sure what this symbol is for

Reply:
@timabell `:invalid_client` is part of open connect/oauth

Reply:
What's the effect of this for the user?